puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/tfm-ror51-release.yaml

```diff
@@ -10,4 +10,4 @@
     dsl:
       !include-raw:
         - pipelines/release/tfmROR51.groovy
-        - pipelines/lib/ansible.groovy
```

Comment:
#485 added a `git.groovy`, mind using it here too?

Reply:
This is the release job which uses a simple git checkout. The PR test job above is using it.